index.html

```diff
@@ -10,21 +10,21 @@
       <div><img src="ga.png" /></div>
       <div class="title">Bank of GA</div>
     </div>
-    
-    <div class="account">
+
+    <div id ="checking" class="account">
       <h2>Checking</h2>
       <div class="balance">$0</div>
-      <input type="text" placeholder="enter an amount" />
-      <input type="button" value="Deposit" />
-      <input type="button" value="Withdraw" />
+      <input class="input" type="text" placeholder="enter an amount" />
+      <input class="deposit" type="button" value="Deposit" />
+      <input class="withdraw" type="button" value="Withdraw" />
     </div>
 
-    <div class="account">
+    <div id="savings" class="account">
       <h2>Savings</h2>
       <div class="balance">$0</div>
-      <input type="text" placeholder="enter an amount" />
-      <input type="button" value="Deposit" />
-      <input type="button" value="Withdraw" />
+      <input class="input" type="text" placeholder="enter an amount" />
+      <input class="deposit" type="button" value="Deposit" />
+      <input class="withdraw" type="button" value="Withdraw" />
     </div>
 
     <script src="atm.js"></script>
```

Comment:
You're going to need to link to jQuery somewhere if any of the code in `atm.js` is going to work!